packages/firestore/android/src/main/java/io/invertase/firebase/firestore/UniversalFirebaseFirestoreModule.java

```diff
@@ -83,4 +83,8 @@
       return null;
     });
   }
+
+  Task<Void> clearPersistence(String appName){
+    return getFirestoreForApp(appName).clearPersistence();
```

Comment:
What happens if this is called after usage of Firestore? It'd be good to see how this is handled.

Reply:
It throws this error: `[firestore/failed-precondition]` as per the documentation: https://firebase.google.com/docs/reference/js/firebase.firestore.Firestore#clearpersistence